### DISCUSSION TITLE
各位大佬，有没有部署的教程？

### DISCUSSION THREAD

**Discussion by chasemoonn** [RealKai42/qwerty-learner#239] [Q&A] [answered]
折腾了一下午 一直出现各种问题

**Comment by chasemoonn** [comment]
![image](https://user-images.githubusercontent.com/46420802/179492463-444dd266-209b-4f7e-a60a-eb2163892a0b.png)

**Comment by RealKai42** [comment, marked as answer]
```
// 安装依赖
yarn
// 打包
yarn build
```
就能在 `build` 文件夹下打包出 html css js 文件，就可以部署到任意 web 服务器中


https://github.com/Kaiyiwing/qwerty-learner/issues/152
也可以参考下这个 issue

**Comment by chasemoonn** [comment]
太感谢您了 成功了





------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年7月18日(星期一) 晚上9:26
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [Kaiyiwing/qwerty-learner] 各位大佬，有没有部署的教程？ (Discussion #239)





 // 安装依赖 yarn // 打包 yarn build  
就能在 build 文件夹下打包出 html css js 文件，就可以部署到任意 web 服务器中
 
#152
 也可以参考下这个 issue

**Comment by SHMoney2021** [comment]
搞定了，搞定了
windows下 powershell安装nodejs, 安装yarn,
计入代码跟目录
yarn
yarn start
多谢前面大佬

**Comment by Mr-VanGogh-K** [comment]
克隆之后，如何再Ubuntu云服务器上部署呢

**Comment by ANKLE999** [comment]
![屏幕截图 2025-04-16 203013](https://github.com/user-attachments/assets/7028bc26-3eba-4c2c-907f-b6cfc45f6615)
求助大佬无法打开
